java/ql/src/Security/CWE/CWE-501/TrustBoundaryViolation.ql

```diff
@@ -0,0 +1,20 @@
+/**
+ * @id java/trust-boundary-violation
+ * @name Trust boundary violation
+ * @description A user-provided value is used to set a session attribute.
```

Comment:
The docs team will probably give better advice after we request a docs review, but normally we should follow the pattern "Syntax X causes behavior Y" for query descriptions. See https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#query-descriptions-description.